src/Core/components/TriggerSet.js

```diff
@@ -124,4 +147,58 @@ export default class triggerSet extends InlineComponent {
       this.coreFunctions.resolveAction({ actionId });
     }
   }
+
+
+  async moveButton({ x, y, z, transient, actionId,
```

Comment:
If there is a reason that it is hard to share this code with the identical copy in UpdateValue, do you want to at least add a comments above each copy pointing to each other, to remind people to keep them in sync?